code/modules/mob/living/bot/bot.dm

```diff
@@ -7,6 +7,8 @@
 	universal_speak = 1
 	density = 0
 
+	makes_dirt = FALSE	// No more dirt from Beepsky
```

Comment:
So _that's_ how one makes mobs not make dirt.

Reply:
Mystery solved.

*x-files*